docs/src/main/tut/docs/overriding-behavior-for-sealed-families.md

```diff
@@ -63,20 +63,21 @@ Then load the config:
 loadConfig[AnimalConf](ConfigFactory.parseString("{ type: Bird, can-fly: true }"))
 ```
 
-With a `CoproductHint` you can even opt not to use any extra field at all. If you encode enumerations using sealed
-traits, you can just write the lowercase name of the class by using an `EnumCoproductHint`. For example, if we create
-an enumeration for seasons:
+If you encode enumerations using sealed traits of case objects, you can use the
+`deriveEnumeration(Reader|Writer|Convert)` methods from the
```

Comment:
Last time we reviewed the documentation on our website, we decided to focus it on the config reading use case (arguably the most important one) and have a dedicated "Config Writers" section to let users know that PureConfig is also able to the reverse operation. It makes the documentation simpler for newcomers and avoids the need for us to always enumerate reader/writer and `ConfigReader`/`ConfigWriter`/`ConfigConvert` on concepts that are clearly extensible to the writing part.

With that in mind, I'd avoid talking about writers here and keep the examples using `deriveEnumerationReader`.

Reply:
Makes sense. I've changed the documentation to only mention the reading use case in b5d0c6a.